envi/__init__.py

```diff
@@ -162,6 +162,23 @@ def archGetRegisterGroups(self):
         allr = [rname for rname in regctx.getRegisterNames()]
         return [('all', allr)]
 
+    def archGetRegisterGroup(self, name):
+        '''
+        Returns a tuple of registers for a named register group.  Returns
+        None if requested name does not exist
+        '''
+        reg_groups = self.archGetRegisterGroups()
+        for gname, group in reg_groups:
```

Comment:
that's a good question.  probably makes more sense to be.  this code is so old i forgot that this wasn't in master already.

Reply:
Hmm, it's generally not a great idea to just swap APIs like this, but I can smoke something like making register groups into a dictionary into my py3 branch (since things are naturally going to break in that anyway)